homeassistant/components/device_tracker/icloud.py

```diff
@@ -78,6 +90,10 @@ def update_icloud(now):
         except PyiCloudNoDevicesException:
             _LOGGER.info('No iCloud Devices found!')
 
+    hass.services.register('device_tracker',
```

Comment:
instead of making a service per device we want 1 service call that uses entity_id to specify which device to target or target all if no entity_id specified.